.github/workflows/docker_publish.yml

```diff
@@ -253,121 +82,52 @@ jobs:
-          ln -s $GITHUB_WORKSPACE /root/build_dir/DAGMC
-
-      - name: Install DAGMC
-        run: |
-          cd $GITHUB_WORKSPACE
-          /bin/bash "CI/scripts/install.sh"
-
-      - name: Run DAGMC Test
-        run: |
-          cd $GITHUB_WORKSPACE
-          /bin/bash "CI/scripts/tests.sh"
-
-  pushing_housekeeping_stable_img:
-    if: ${{ github.repository_owner == 'svalinn' }}
-    needs: [test_img]
-    runs-on: ubuntu-latest
-    env:
-      hdf5_versions: 1.10.4
-      hdf5_build_dir: hdf5_build_dir
-    strategy:
-      matrix:
-        ubuntu_versions : [18.04,]
-
-    name: Housekeeping':' latest -> stable
-    steps:
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
+        static_exe : [
+         OFF,
+         ON,
+        ]
 
+    name: Installing DAGMC
+    steps:        
       - name: Log in to the Container registry
-        uses: docker/login-action@v1
+        uses: docker/login-action@v2
         with:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Push Image as stable img
-        uses: akhilerm/tag-push-action@v1.0.0
-        with:
-          src: ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}-housekeeping:ci_testing
-          dst: ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}-housekeeping:stable
+      - name: Checkout repository
+        uses: actions/checkout@v3
 
-      - name: Push Image as latest img
-        uses: akhilerm/tag-push-action@v1.0.0
+      - name: Test DAGMC in Docker image
+        uses: firehed/multistage-docker-build-action@v1
         with:
-          src: ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}-housekeeping:ci_testing
-          dst: ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}-housekeeping:latest
-
-  pushing_test_stable_img:
-    if: ${{ github.repository_owner == 'svalinn' }}
-    needs: [test_img]
-    runs-on: ubuntu-latest
-    env:
-      hdf5_versions: 1.10.4
-      hdf5_build_dir: hdf5_build_dir
-
-    strategy:
-      matrix:
-        ubuntu_versions : [
-          18.04,
-          20.04,
-          ]
-        compiler : [
-          gcc,
-          clang,
-          ]
-        hdf5_versions : [
-         1.10.4,
-        ]
-        moab_versions : [
-         5.3.0,
-        ]
-
-    name: Ubuntu${{ matrix.ubuntu_versions }} + ${{ matrix.compiler }} hdf5 ${{ matrix.hdf5_versions }} + Moab ${{ matrix.moab_versions }}':' latest -> stable
-
-    steps:
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
+          repository: ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}-${{ matrix.compiler}}-ext-hdf5_${{ matrix.hdf5_versions}}-moab_${{ matrix.moab_versions }}
+          stages: base, external_deps, hdf5, moab, dagmc
```

Comment:
We don't need to explicitly build all these stages at this point.  Most were just built anyway.  This action should use the container repository to blow past these anyway, but for maintainability I recommend we remove all except `dagmc`

Reply:
resolved by removing all but dagmc from the stages